tests/spec/core/xref-spec.js

```diff
@@ -418,4 +456,113 @@ describe("Core — xref", () => {
       "[html],[infra],[local-1],[local-2],[local-3],[local-4],[webidl]"
     );
   });
+
+  describe("inline IDL references", () => {
+    it("ignores inlines starting with backslash", async () => {
+      const body = `<section><p id="test">{{{\\PASS }}}</p></section>`;
+      const config = { xref: { url: apiURL } };
+      const ops = makeStandardOps(config, body);
+      const doc = await makeRSDoc(ops);
+      const el = doc.getElementById("test");
+      expect(el.querySelector("code a")).toBeFalsy();
+      expect(el.textContent).toEqual("{{{PASS}}}");
+    });
+
+    it("links inline IDL references", async () => {
+      const body = `
+      <section id="test">
+        <p id="link1">{{{ Window }}}</p>
+        <p id="link2">{{{ [[query]] }}}</p>
+        <p id="link3">{{{ [[type]] }}} is ambiguous.</p>
```

Comment:
type is an internal slot, no? so it shouldn't be ambiguous because it can only link internally?

Reply:
We are yet to do that.
Presently  (as discussed on Slack) - I was expecting `{{{ }}}` to work only for external IDL references. But we plan to support them locally also. (First look locally, if not found look external)

Unless: I'm missing some fact like "internal slots cannot be external references"

Reply:
Internal slots can’t be external references :)

Reply:
So, does this make sense?
``` html
<!-- external - link both Credential and [[CollectFromCredentialStore]] -->
<p>{{{ Credential.[[CollectFromCredentialStore]](options, sameOriginWithAncestors) }}}</p>

<!-- local -->
<p>{{{ [[CollectFromCredentialStore]](options, sameOriginWithAncestors) }}}</p>
```